Userland/Utilities/cal.cpp

```diff
@@ -41,9 +44,7 @@ static ErrorOr<Vector<String>> month_lines_to_print(int month, int year)
             row.append("   "sv);
         } else {
             if (year == current_year && month == current_month && day_to_print == current_day) {
-                // FIXME: To replicate Unix cal it would be better to use "\x1b[30;47m%2d\x1b[0m " in here instead of *.
-                //        However, doing that messes up the layout.
-                row.appendff("{:02}*", day_to_print);
+                row.appendff(INVERT_OUTPUT "{:02}" RESET_OUTPUT " ", day_to_print);
```

Comment:
The `ANSI_` prefix got lost here?

also, might as well just inline that `\x1b[7m` and comment on it instead?

Reply:
Fixed the prefix, thanks!

>also, might as well just inline that \x1b[7m and comment on it instead?
 
I don't know about this - for me it's much more readable like that. Also, `ttfdisasm` does it this way as well :D